tests/foreman/cli/test_settings.py

```diff
@@ -259,7 +261,10 @@ def test_positive_update_email_reply_address(setting_update):
     email_reply_address = Settings.list(
         {'search': 'name=email_reply_address'}, output_format='json'
     )[0]
-    assert email_reply_address['value'] == email_address
+    updated_email_address = (
```

Comment:
```suggestion
updated_email_address = str(email_reply_address['value']).replace('"', r'\"').replace('`', r'\`')
```

Reply:
the round brackets are added after executing `pre-commit run`